cmake/OpenCVDetectWebGPU.cmake

```diff
@@ -0,0 +1,24 @@
+if(HAVE_WEBGPU OR WITH_WEBGPU)
+  set(WEBGPU_INCLUDE_DIRS "${PROJECT_SOURCE_DIR}/3rdparty/include/webgpu/include")
+  set(WEBGPU_LIBRARIES "${PROJECT_SOURCE_DIR}/3rdparty/include/webgpu/lib")
+endif()
+
+if(HAVE_WEBGPU OR WITH_WEBGPU)
+  include_directories(${WEBGPU_INCLUDE_DIRS})
+  link_directories(${WEBGPU_LIBRARIES})
+  link_libraries(dawn_proc dawn_native dawn_wire)
+
+endif()
```

Comment:
if (HAVE_WEBGPU)
......
endif()
and put these codes after setting HAVE_WEBGPU